Turbo Sort TSORT.c

```diff
@@ -5,7 +5,7 @@ int arr[1000000]={0};
      
 void main()
 {
-	int a, i,n;
+	int a,n;//See here in this code i is useless so i think that can be deleted.
```

Comment:
No need to add this comment here. add this comment to the pull request instead.

Reply:
Yes yes...